airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/parquet/ParquetSerializedBuffer.java

```diff
@@ -101,6 +101,11 @@ public long accept(final AirbyteRecordMessage record) throws Exception {
     }
   }
 
+  @Override
+  public long accept(final String recordString, final long emittedAt) throws Exception {
```

Comment:
same note here on the AvroRecordFactory benefiting from our interface simplification.

Reply:
one more place to throw an unsupported operation exception.